containerization-installs/docker-install.sh

```diff
@@ -28,7 +28,7 @@ fi
 # Creating a keyrings directory 
 if (test -d /etc/bin/keyrings)
 then
-  echo "creating keyrings"
+  echo "keyrings directory already exist"
```

Comment:
Looks good to me.  For the sake of being 
pedantic, might as well change the comment in the else statement as well....